.github/workflows/ci.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     - name: Set up Zig
       uses: korandoru/setup-zig@v1
       with:
-        zig-version: 0.10.1
+        zig-version: master
```

Comment:
This really isn't great, because zig changes a lot and so from one CI run to the next, we could end up with something that worked and no longer does, which would be confusing as the zig version isn't displayed. I'll make sure it does, so that we can live with this for the time being.